index.html

```diff
@@ -2,12 +2,67 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>Quantified Self - Diary</title>
-</head>
-<body>
+  <title>Coat or No Coat</title>
+  <link rel="stylesheet" type="text/css" href="lib/styles/reset.css">
+  <link rel="stylesheet" type="text/css" href="lib/styles/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Hanalei+Fill|Patrick+Hand" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Text+Me+One" rel="stylesheet">
+
+  <body>
+
+  <nav class="nav">
+    <ul class="nav-container">
+      <!-- <div class="logo-container"> -->
+        <li><a href="" class="logo" id="logo-pic"><img src="lib/CONC-logo.jpg"></a></li>
+        <li><a href="" class="logo" id="logo-headline">Coat or No Coat</a></li>
+      <!-- </div> -->
+      <!-- <div class="nav-links-container"> -->
+      <li><a href="" class="nav-links" id="login-logout">Log Out</a></li>
+        <li><a href="" class="nav-links" id ="about">About</a></li>
+        <li><a href="" class="nav-links" id ="my-account">My Account</a></li>
+      <!-- </div> -->
+    </ul>
+  </nav>
+
+  <div class="body-containers">
+    <div class="info-container">
+      <h1 class="headline">Today's Coat</h1>
+      <h1 id="pink">Fall Coat</h1>
+    </div>
+    <div class="between"></div>
+
+    <div class="info-container">
+      <h1 class="headline">My Coats</h1>
+        <div class="coat-container">
```

Comment:
I have an idea on how to make responsive clouds, like we were talking about earlier. Got an idea from [here](https://stackoverflow.com/questions/30919053/creating-a-responsive-cloud-shape), and more understanding from [here](https://www.w3schools.com/graphics/svg_path.asp). Just an idea in case there's time to explore it. 

Maybe something like this: 
*index.html*
```
<svg viewBox='0 0 105 105' >
      <path d='M 25,60
               a 20,20 1 0,0 0,40
               h 50
               a 20,20 1 0,0 0,-40
               a 10,10 1 0,0 -15,-10
               a 15,15 1 0,0 -35,10
               z' />
        <text x="35" y="90" font-size="35" fill="#94958B">hi</text>
</svg>
```
*styles.css*
```
svg {
  height: 100%;
  width: 100%;
}
path {
  fill: white;
  stroke: #f3f3f2;
  stroke-width: .5;
}
path:hover {
  fill: #f3f3f2;
  stroke: #94958B;
}
```